exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,8 @@
 // Start by creating a variable `message`
+var greetingStart = "Hello,";
+var name = "my name is jaspreet";
+
+var message = greetingStart + name;
```

Comment:
be careful of spaces when you do `string1 + string2` as it may look like `HelloWorld`